atlasexec/atlas.go

```diff
@@ -335,8 +335,9 @@ func (c *Client) SchemaInspect(ctx context.Context, params *SchemaInspectParams)
 	if params.DevURL != "" {
 		args = append(args, "--dev-url", params.DevURL)
 	}
-	if params.Format == "sql" {
-		args = append(args, "--format", "{{ sql . }}")
+	if params.Format != "" {
+		format := fmt.Sprintf("{{ %s . }}", params.Format)
```

Comment:
should we sanitize against template injection? 🤓

Reply:
No, we use `text/template`.